docs/modules/ROOT/pages/release-notes/release-notes-2021-7.adoc

```diff
@@ -0,0 +1,29 @@
+= Release Notes - Payara Platform Community 5.2021.7
+
+== Supported APIs and Applications
+
+* Jakarta EE 8
+* Java EE 8 Applications
+* Jakarta EE 9
+* MicroProfile 4.1
+
+=== New Features
+* [https://github.com/payara/Payara/pull/5398[FISH-5646]] Add Client Certificate Validation API
+
+=== Bug Fixes
+* [https://github.com/payara/Payara/pull/5399[FISH-5711]] Fix Could Not Load Formatter Class Error in Payara Micro
+* [https://github.com/payara/Payara/pull/5397[FISH-5701]] Application does not deploy anymore on Payara Server Docker image
+* [https://github.com/payara/Payara/pull/5374[FISH-5695]] ArrayIndexOutOfBoundsException when the ConfigProperty value ends in a dollar sign \[Community Contribution\]
```

Comment:
A community contribution should be in () and not in \[\] - github interprets \[ as ( but Antora doesn't. And with the github user ID of the committer, and in the beginning of the description. So like this:
```suggestion
* [https://github.com/payara/Payara/pull/5374[FISH-5695]] (Community Contribution - https://github.com/Empressia[Empressia]) ArrayIndexOutOfBoundsException when the ConfigProperty value ends in a dollar sign (Community Contribution
```
So that it's similar to how we did it in the past, for example in https://github.com/payara/Payara-Community-Documentation/blob/master/docs/modules/ROOT/pages/release-notes/release-notes-2021-4.adoc